### PR TITLE
[WIP] add end to end test

### PR DESCRIPTION
**Pull request by linclark**
I ran into a few problems when trying to add an end to end test.
1. waitForMessages doesn't seem to be picking up on the new console. We're still rendering (and hiding) the previous console, and it seems to pick up those messages
2. I couldn't get the system to pick up on the new TEST_URL file I added, test-console-log.html

### CODE REVIEW

File: devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js [hunk 1]
```diff
@@ -31,7 +31,10 @@ function ConsoleApiCall(props) {
     repeat
   ];
 
-  return dom.div({ className: "message cm-s-mozilla" },
+  // @TODO Use of "is" is a temporary hack to get the category and severity
+  // attributes to be applied. I believe these are only used for testing so
+  // can probably eventually be removed.
+  return dom.div({ is: "fdt-message", category: message.category, severity: message.severity },
```
Comment (bgrins) [inline devtools/client/webconsole/new-console-output/components/message-types/console-api-call.js]: Will need to include `"class": "message cm-s-mozilla"` to get waitForMessages working

**Comment by bgrins** [comment]
Please change head.js as such: https://github.com/bgrins/gecko-dev/commit/163aa9ac69d036583df1c3b9d65edf4b2cdca328#diff-172ca1b6f589eec7862cd98f68763e55R1394

**Comment by bgrins** [comment]
Instead of 'repeats: 2' here it should be `text: 'foo'` at least, to get it working for now

**Comment by linclark** [comment]
I'm going to close this one out because it got messed up due to some faulty rebasing early in the project. The commit that actually introduces the test is 1ec8f88bc74

**Pull request closed without merging**